var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -73,6 +74,8 @@ class Dealii(CMakePackage, CudaPackage):
             description='Compile with Arpack and PArpack (only with MPI)')
     variant('adol-c',   default=True,
             description='Compile with ADOL-C')
+    variant('cgal',     default=True,
+            description='Compile with CGAL')
```

Comment:
Variant can also be conditional:
```suggestion
    variant('cgal', default=True, when='@9.4:',
            description='Compile with CGAL')
```

Reply:
Ah, great! Thanks for the suggestion. I don't think that I (we?) knew about this. Maybe I should update the other variants in a follow up PR.